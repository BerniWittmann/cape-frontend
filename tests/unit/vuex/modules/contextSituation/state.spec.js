import cSState from '@/vuex/modules/contextSituation/state'

describe('Vuex', () => {
  describe('Modules', () => {
    describe('contextSituation', () => {
      describe('State', () => {
        it('has an empty base state', () => {
          expect(cSState).toMatchSnapshot()
        })
      })
    })
  })
})
