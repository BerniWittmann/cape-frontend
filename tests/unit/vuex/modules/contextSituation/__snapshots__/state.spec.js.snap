// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vuex Modules contextSituation State has an empty base state 1`] = `
Object {
  "activeContextSituation": undefined,
  "contextSituations": Array [],
}
`;
